hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -399,9 +415,12 @@ private class Emit[C](
     def emitStream(ir: IR, mb: EmitMethodBuilder[C] = mb): COption[EmitStream.SizedStream] =
       EmitStream.emit(this, ir, mb, region, env, container)
 
-    def emitVoid(cb: EmitCodeBuilder, ir: IR, mb: EmitMethodBuilder[C] = mb, region: Value[Region] = region, env: E = env, container: Option[AggContainer] = container, loopEnv: Option[Env[LoopRef]] = loopEnv): Unit =
+    def emitVoid(ir: IR, cb: EmitCodeBuilder = cb, mb: EmitMethodBuilder[C] = mb, region: Value[Region] = region, env: E = env, container: Option[AggContainer] = container, loopEnv: Option[Env[LoopRef]] = loopEnv): Unit =
```

Comment:
This is really a comment about the top-level `emit` methods, and maybe is out of scope for this PR. With an `EmitCodeBuilder` as an argument, do we also need an `EmitMethodBuilder`?

Reply:
As long as we can get the EmitMethodBuilder off of the emit code builder, probably not.